src/lightning_app/core/app.py

```diff
@@ -45,6 +46,7 @@ class LightningApp:
     def __init__(
         self,
         root: "lightning_app.LightningFlow",
+        default_cloud_compute: Optional["lightning_app.CloudCompute"] = None,
```

Comment:
should this be named `default_**`. What does `default` indicate which is not clear from just using `cloud_compute`?

Reply:
@tchaton @awaelchli

Reply:
@tchaton chose this name. As I mentioned, there would be logic needed that applies the value given here to all Works that have no user-provided compute configuration. If this logic is not intended to be added, then we should not call it `default_...`, and `flow_cloud_compute` instead as proposed above.

Reply:
@tchaton @awaelchli any objections if I revert the change to `flow_cloud_compute` and we address the default behaviour separately?

Reply:
No objection. @tchaton ?

Reply:
I like `flow_cloud_compute`, it’s more specific compared to `default`

Reply:
:+1: renamed back to `flow_cloud_compute`.